### ISSUE TITLE
Why is the object color so strange after the geometry stage?

### ISSUE THREAD

**Issue by DaLi-Jack** [modelscope/richdreamer#55]
Hi! Thanks for your great work!
I am currently testing your NeRF-based code. However, I found that even if I add RGB prior in the geometry stage, although the geometry of the object is good, the color of the object obtained using your method is still very strange.
Specific experimental settings are as follows: 
I only use the geo-fast stage according to [geo.yaml](https://github.com/modelscope/richdreamer/blob/main/configs/nd-mv-nerf/geo.yaml), and I change [L91](https://github.com/modelscope/richdreamer/blob/d60b3c079d9d7bf342039bf4f9bd52bf33b8f9e1/configs/nd-mv-nerf/geo.yaml#L91) "comp_normal_cam_vis" to "comp_rgb", which means I all use RGB prior. And I use "A chair" as the prompt. 
The experimental result is as follows:
![it001200-0000](https://github.com/user-attachments/assets/277d2539-a788-4ef1-8389-653365181f0e)
Have you observed similar experimental results, and do you have any good solutions? 
Any help would be greatly appreciated!

**Comment by lingtengqiu** [comment]
NeRF-based method produce soft-geometry, so the right side of your figure seems to be strange.

**Comment by gxd1994** [comment]
@DaLi-Jack 
 We decompose 3D generation into two parts: geometry generation and appearance generation.  In the geometry generation stage(geo.yaml),  we focus on generating geometry by using the normal-depth diffusion model. we also use rgb diffusion model but with a smaller loss weight(1.0 vs 10.0 in [L122](https://github.com/modelscope/richdreamer/blob/d60b3c079d9d7bf342039bf4f9bd52bf33b8f9e1/configs/nd-mv-nerf/geo.yaml#L122)). 
 If you want to get good color,  you can run  appearance generation (scripts in [L37)](https://github.com/modelscope/richdreamer/blob/d60b3c079d9d7bf342039bf4f9bd52bf33b8f9e1/scripts/nerf/run_batch_fast.sh#L37)

**Comment by gxd1994** [comment]
@DaLi-Jack 
We decompose 3D generation into two parts: geometry generation and appearance generation. In the geometry generation stage(geo.yaml), we focus on generating geometry by using the normal-depth diffusion model. we also use rgb diffusion model but with a smaller loss weight(1.0 vs 10.0 in [L123-124](https://github.com/modelscope/richdreamer/blob/d60b3c079d9d7bf342039bf4f9bd52bf33b8f9e1/configs/nd-mv-nerf/geo.yaml#L123)).
If you want to get good color, you can run appearance generation (scripts in [L37)](https://github.com/modelscope/richdreamer/blob/d60b3c079d9d7bf342039bf4f9bd52bf33b8f9e1/scripts/nerf/run_batch_fast.sh#L37)

**Comment by DaLi-Jack** [comment]
Yes, I have tried the texture optimization stage for the mesh. After that, the appearance is excellent, as shown in the following image:
![image](https://github.com/user-attachments/assets/e2e9b3e2-6328-4870-be50-c19bbd7c8aee)
However, in my case, I didn't use the normal-depth diffusion model. I only used the stable diffusion model with RGB prior. I wonder why the color is so strange. Because I originally thought that with RGB prior added, at least reasonable colors should be generated and it shouldn't be so strange.
